src/state-incentive-details.tsx

```diff
@@ -301,7 +306,9 @@ const renderCardCollection = (incentives: Incentive[]) => (
     {incentives
       // Put IRA rebates after everything else
       .sort((a, b) => +isIRARebate(a) - +isIRARebate(b))
-      .map((incentive, index) => renderIncentiveCard(index, incentive))}
+      .map((incentive, index) => (
+        <IncentiveCard key={index} incentive={incentive} />
```

Comment:
I know you're not supposed to use a numerical index as a key, but incentives don't have identifiers for now

Reply:
I'd be in favor of returning one!